packages/picasso-lab/src/DrilldownMenu/DrilldownMenu.tsx

```diff
@@ -0,0 +1,37 @@
+import { BaseProps, CompoundedComponentWithRef } from '@toptal/picasso-shared'
+import SelectList, { SelectListAttributes } from '@toptal/picasso/SelectList'
+import React, { forwardRef, ReactNode, useMemo, useState } from 'react'
+
+import DrilldownMenuItem from '../DrilldownMenuItem'
+import DrilldownMenuContext from './DrilldownMenuContext'
+
+export interface Props extends BaseProps, SelectListAttributes {
+  /* Menu items */
+  children?: ReactNode
+}
+
+export interface StaticProps {
+  Item: typeof DrilldownMenuItem
+}
+
+export const DrilldownMenu = forwardRef<HTMLUListElement, Props>(
+  function Drilldown (props, ref) {
+    const { className, style, children, ...rest } = props
+    const [activeMenuKey, setActiveMenuKey] = useState<string>()
+    const context = useMemo(() => ({ activeMenuKey, setActiveMenuKey }), [
```

Comment:
It's not a context, actually. It's a context value. 🙂

```suggestion
    const contextValue = useMemo(() => ({ activeMenuKey, setActiveMenuKey }), [
```

Reply:
Fixed